packages/errors/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@vercel/errors",
```

Comment:
Lets call this `@vercel/error-utils`

I'm also wondering if you plan to  import it from builders. In which case, it should probably go in `@vercel/build-utils` instead.

Reply:
Hmm I could move it to `@vercel/build-utils`. It is going to be used in `@vercel/fs-detectors` next (https://github.com/vercel/vercel/pull/8742#discussion_r1006370174)